src/core/Formatter.ts

```diff
@@ -53,10 +56,27 @@ export default class Formatter {
 
   private formatAst(statements: Statement[]): string {
     return statements
-      .map(stat => new StatementFormatter(this.cfg, this.params).format(stat))
+      .map(stat => this.formatStatement(stat))
```

Comment:
can be `.map(this.formatStatement)`

Reply:
Won't work. `this` context is needed inside `formatStatement`. I could do:

```js
.map(this.formatStatement, this)
```

but that's more of a question with style. I would have preferred the separate `this` argument before arrow functions arrived to JavaScript, but nowadays IMHO the arrow-function is preferable as it will also work in functions that don't accapt a separate `this` parameter.